src/test/java/org/elasticsearch/common/logging/log4j/LoggingConfigurationTests.java

```diff
@@ -140,7 +140,7 @@ private static File resolveConfigDir() throws Exception {
         return new File(url.toURI());
     }
 
-    private static String loggingConfiguration(String suffix) {
+    protected static String loggingConfiguration(String suffix) {
```

Comment:
Had to make this method protected as it is being used within the `testResolveYamlLoggingConfig()`